Gems/Atom/Feature/Common/Assets/Shaders/Shadow/ClearShadowmap.azsl

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
```

Comment:
Needed?

Reply:
over-aggressive copy-pasting of copyright header :D. Will fix.